README.md

```diff
@@ -11,13 +11,30 @@ This repository hosts the documentation for [n8n](https://n8n.io/), an extendabl
 
 * Python 3.8 or above
 * Pip
-* n8n recommends using a virtual environment when working with Python, such as [venv](https://docs.python.org/3/tutorial/venv.html).
 * Follow the [recommended configuration and auto-complete](https://squidfunk.github.io/mkdocs-material/creating-your-site/#minimal-configuration) guidance for the theme. This will help when working with the `mkdocs.yml` file.
 * The repo includes a `.editorconfig` file. Make sure your local editor settings **do not override** these settings. In particular:
 	- Don't allow your editor to replace tabs with spaces. This can affect our code samples (which must retain tabs for people building nodes).
 	- One tab must be equivalent to four spaces.
+* n8n recommends using a virtual environment when working with Python, such as [venv](https://docs.python.org/3/tutorial/venv.html).
+
+To create a virtual environment, type:
+
+```plaintext
+python -m venv venv
+```
+
+To activate it, on Windows, run:
 
-### Steps
+```plaintext
+venvScriptsactivate
```

Comment:
This is incorrect, at least in PowerShell:

```shell
.\venv\Scripts\activate.ps1
```

^ that would work

This gets into the wider problem though. Everyone is on different environments. We already recommend using venv, and expect people to follow the link in the recommendation and do the appropriate setup for their system.